packages/terra-card/src/CardBody.scss

```diff
@@ -9,4 +9,10 @@
     padding-left: var(--terra-card-padding-left, 10px);
     padding-right: var(--terra-card-padding-right, 10px);
   }
+
+  .center-content {
+    align-items: center;
+    display: flex;
+    justify-content: center;
```

Comment:
It may be worth discussing if there should be a distinction between centering content vertically, horizontally, or both unless the only use-case is always centering content vertically and horizontally.

Reply:
I would think both vertically and horizontally for now. If we run into scenarios that need either, we can revisit?!

Reply:
I'm curious if vertically centering the content with `align-items: center;` has any effect if the Card.Body height is always auto.

Reply:
Because the Card.Body accepts multiple children it's worth mentioning `display: flex` has more side effects than just centering content. This may not be a huge issue if the intent is to always center a single child and it's opt in.

```
<Card>
  <Card.Body>
    <div>Block 1</div>
    <div>Block 2</div>
  </Card.Body>
</Card>;
```

Before:
![image](https://user-images.githubusercontent.com/6720991/36761155-6ad69842-1be3-11e8-9e64-4230bb2b5b59.png)
With `isContentCentered`:
![image](https://user-images.githubusercontent.com/6720991/36761185-8967e252-1be3-11e8-823d-468fd0a817a6.png)

Reply:
As @StephenEsser  noted, I don't think vertical-align makes much since because of how the height is calculated.

The blocks sitting inline like that seems like an issue though. It could be mitigated by conditionally wrapping the children in another div when the centering is used
https://codepen.io/anon/pen/MQZpaE
Multiple items is a definite use case, and forcing the consumer to put everything in a div seems bad

Reply:
Good point @StephenEsser I feel like there should be a distinction between horizontal and vertical alignment because of the side effect you mentioned but I don't have lot of experience to comment on if our use cases have been to center the contents horizontally and not vertically in which case I can just remove the vertical alignment and this component can be updated later when use case arises for vertical alignment.

If we decide to take in separate props I think we can create `isContentCenteredHorizontally` and `isContentCenteredVertically`. 

@bjankord @alex-bezek thoughts ?

Reply:
I vote to split out the props.

Reply:
With the Card.Body always having a height of auto I don't think there is a concept of vertical centering. The content is always the full height of Card.Body ( minus padding ), so there are no adjustments to be made above or below the content. In order to center the content the Card.Body would need to be taller than the content.

Reply:
> With the Card.Body always having a height of auto I don't think there is a concept of vertical centering. 

Agreed, let's drop the `align-items: center;` style. I'm also open to renaming the prop to `isContentCenteredHorizontally` or just leaving it as `isContentCentered`.

> it's worth mentioning display: flex has more side effects than just centering content. This may not be a huge issue if the intent is to always center a single child and it's opt in.

Reply:
Theres a few different ways I can think of to center items

Reply:
I'm not really sure we want to get in the game of just providing react wrappers of different css styles do we? 
I feel like the problem we are trying to solve with this isn't "the ability to center things" but rather that I have found a common pattern in our callouts for centering the contents of a card.

We try to treat `any css specific to a page as a code smell` since it signifies something not buildable with our existing component library, but a concept that we need to deliver for UX. So if its a unique situation for our app, we just try to make it a re-usable component in our application. I'm almost against the center component just because it prevents us from easily finding those `css files that just consist of a single text-align: center class and that's it` and identifying that pattern.

For reference where this is coming from, I noticed that in our app, over the last couple months (and including the page @vaibhavpaliwal is working on) there have been
* 12 instances of `text-align: center;` added
* 4 are the only class/style in their components css
* 9 are in cards

We only have 1 area we added flex box. My understanding is that flex box applies to all the contents, but text-align center only affects non block level elements. I understand others may have other needs, but our stuff based off existing UX callouts hasn't encountered the issue of needing to center block elements.

I think the 3 options are
* Just use text-align. It solves the 90% use case, and consumer that actually have content with block items can make that center itself and put it in the card. https://codepen.io/anon/pen/QQYvBV
* Use the idea of throwing an extra div in there to fix flex box issue https://codepen.io/anon/pen/YeBVMQ
* Not allow cards to center their body, but just allow consumers to center their own contents. Provide a center component.

Reply:
I'm good with having the `isContentCentered` just set `text-align: center;`. I agree that for cases this style doesn't center stuff, the consumer can make that content / component center itself and put it in the card.

Reply:
updated to use `text-align: center;`
Thanks!